Lib/test/_test_multiprocessing.py

```diff
@@ -4898,8 +4898,14 @@ def _test_dict(cls, obj):
         assert len(obj) == 1
         assert obj['foo'] == 5
         assert obj.get('foo') == 5
-        # TODO: fix https://bugs.python.org/issue35918
-        # assert obj.has_key('foo')
+        # See https://bugs.python.org/issue35918
+        try:
+            # this is expected to fail
+            obj.has_key('foo')
+        except AttributeError:
+            pass
+        else:
+            assert "obj.has_key should have raised AttributeError"
```

Comment:
I think you can simply remove this test.

Reply:
Thanks